pytube/cipher.py

```diff
@@ -273,7 +273,16 @@ def get_throttling_function_name(js: str) -> str:
         function_match = regex.search(js)
         if function_match:
             logger.debug("finished regex search, matched: %s", pattern)
-            return function_match.group(1)
+            function_name = function_match.group(1)
+            is_Array = True if '[' or ']' in function_name else False
```

Comment:
This is wrong, as pointed out by [this comment](https://github.com/pytube/pytube/issues/1243#issuecomment-1037010471).